CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# v0.0.5
```

Comment:
```suggestion
# Unreleased
```

This version already occupied. 

There is a github releases used for tracking changes.